lib/datadog/core/telemetry/schemas/base/v1/application.rb

```diff
@@ -0,0 +1,31 @@
+module Datadog
+  module Core
+    module Telemetry
+      module Schemas
+        module Base
+          module V1
+            # Describes attributes for application environment object
+            class Application
+              attr_reader :language_name, :language_version, :service_name, :tracer_version, :env, :runtime_name,
+                          :runtime_patches, :runtime_version, :service_version, :products
+
+              def initialize(language_name, language_version, service_name, tracer_version, env = nil, runtime_name = nil,
```

Comment:
This argument list might be more appropriate as keyword arguments.

https://thoughtbot.com/blog/ruby-2-keyword-arguments